polly/Basic.py

```diff
@@ -48,7 +48,7 @@ def create(
         Args:
              display_name (str): display name of the omixatlas.
              description (str): description of the omixatlas.
-             repo_name (str): repo_name which is used to create index in db.
```

Comment:
@shreya-el create function in OA takes only repo_name.

Reply:
resolved